app/src/main/java/de/westnordost/streetcomplete/quests/segregated/AddSegregated.java

```diff
@@ -0,0 +1,67 @@
+package de.westnordost.streetcomplete.quests.segregated;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.meta.OsmTaggings;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.powerpoles_material.AddPowerPolesMaterialForm;
+
+public class AddSegregated extends SimpleOverpassQuestType {
+
+	@Inject
+	public AddSegregated(OverpassMapDataDao overpassServer) {
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters() {
+		return
+			"ways with " +
+				"(" +
+				"(highway = path and bicycle = designated and foot = designated)" +
```

Comment:
> BTW, did you consider what happens if this quest get's tagging invalid values? Such as highway = footway and foot = no (and e.g. bicycle=designated to trigger this quest)?

No, but I will do it now. A good point, thanks for catching.

------

Such combination appears 25 times worldwide ( http://overpass-turbo.eu/s/Aj7 ) so I will happily ignore it.

Reply:
Of course it is rare – everything else would be embarrassing. :wink: 
But I don't think ignoring is the best way here, because we are not talking about including tags/quests, because they are rare, but about a mistagging that may affect this quest.

---

Thinking about it, however, I guess it really is not bad to show this quest for such situations. Because it does not worsen the mistagged information (i.e. now the footway/foot=no is tagged with segregated=no or so, well… who cares?) and one also does not expect that SC would fix these issues.

----

Generally maybe it would be a good idea to add these two _other answers_:
> "This is a cycleway only" | "This is a footway only"

Just leaving a note for these small changes, which are very easy to identify for a surveyor looking at the path, would be a bit too much, IMHO.

Reply:
Regarding your last suggestion, @rugk , if you look outside "signposted" Germany, you will find that many sidewalks and paths will have no signs at all. How would one distinguish the case that pedestrians and cyclists are mixed from that no cyclists are allowed (i.e. because surveyor doesn't see one right now or subjectively thinks cyclists should not use this path)?

Reply:
Oh, yeah, indeed difficult. So rather leave them out.